WhaleSpotting/Models/DbModels/WhaleSpottingContext.cs

```diff
@@ -7,7 +7,7 @@ public class WhaleSpottingContext: DbContext
         public WhaleSpottingContext(DbContextOptions<WhaleSpottingContext> options) : base(options)
         {
         }
-        
         public DbSet<TestDbModel> Test { get; set; }
```

Comment:
We can remove the test model now we have actual migrations

Reply:
Comment addressed.